extensions/blocks/conversation/components/participants-controls.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	DropdownMenu,
+	TextControl,
+	BaseControl,
+	Button,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+function AddParticipantButton( {
+	className,
+	onAdd,
+	participants = [],
+} ) {
+	return (
+		<BaseControl>
+			<div className={ `${ className }__participant` }>
+				<Button
+					className={ `${ className }__add-button` }
+					label={ __( 'Add Participant', 'jetpack' ) }
+					onClick={ () => onAdd( `Participant ${ participants.length + 1 }` ) }
+					isSecondary
+					isSmall
+				>
+					{ __( 'Add participant', 'jetpack' ) }
+				</Button>
+			</div>
+		</BaseControl>
+	);
+}
+
+function ParticipantsLabelControl( {
+	className,
+	participants,
+	onChange,
+	onDelete,
+} ) {
+	return (
+		<Fragment>
+			{ map( participants, ( { participant, participantSlug } ) => (
+				<BaseControl className={ `${ className }__participant-control` }>
```

Comment:
There's a warning coming from this function about a missing `key` prop

<img width="809" alt="image" src="https://user-images.githubusercontent.com/1699996/101435415-e9fe2780-38d1-11eb-840f-353a309e02c7.png">

Reply:
Fixed https://github.com/Automattic/jetpack/pull/17916/commits/e1ab5ccbb11997657edcb64a6d4e95b8fa816626. Could you confirm it and mark it as resolved? Thanks in advance.

Reply:
Confirmed that the `key` warning is fixed now.